app/src/full/java/io/homeassistant/companion/android/sensors/LocationSensorManager.kt

```diff
@@ -996,14 +983,20 @@ class LocationSensorManager : BroadcastReceiver(), SensorManager {
 
         var geofenceCount = 0
         getEnabledServers(latestContext, zoneLocation).map { serverId ->
-            ioScope.async {
+            sensorWorkerScope.async {
```

Comment:
I am not sure if this shouldve been updated, removed or reverted. For consistency sake I kept it as its a `async` call and just applied our worker to it.

Reply:
Why would you change this to run this on the UI thread instead of a worker thread?